src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenConfigs.js

```diff
@@ -0,0 +1,140 @@
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ * Adyen Salesforce Commerce Cloud
+ * Copyright (c) 2022 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+const dwsystem = require('dw/system');
+const adyenCurrentSite = dwsystem.Site.getCurrent();
+
+// Retrieve a custom preference
+function getCustomPreference(field) {
+    let customPreference = null;
+    if (adyenCurrentSite && adyenCurrentSite.getCustomPreferenceValue(field)) {
+        customPreference = adyenCurrentSite.getCustomPreferenceValue(field);
+    }
+    return customPreference;
+}
+
+const adyenConfigsObj = {
+    // Get the current adyen environment mode (live or test)
```

Comment:
Should we create consistency with adding/removing the comments for each config?

Reply:
good point! I removed them since the function names are descriptive enough. Plus we already have description in metadata.